test_scripts/Resumption/InteriorVehicleData/021_InteriorVD_resumption_in_3rd_ignition_cycle.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL resumes interior vehicle data in 3rd ignition cycle
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1 via GetInteriorVehicleData(module_1)
+--
+-- Sequence:
+-- 1. IGN_OFF and IGN_ON are performed
+-- 2. IGN_OFF and IGN_ON are performed
```

Comment:
@GetmanetsIrina I would propose next simplification
```suggestion
-- 1. Two IGN_OFF and IGN_ON cycles are performed
```
I propose similar simplification for three IGN_OFF cycle for next test script.